math/pi/montecarlopi.go

```diff
@@ -0,0 +1,27 @@
+// montecarlopi.go
+// description: Calculating pi by the Monte Carlo method
+// details:
+// implementation of Monte Carlo Algorithm for the calculating of Pi - [Monte Carlo method](https://en.wikipedia.org/wiki/Monte_Carlo_method)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see montecarlopi_test.go
+
+package pi
+
+import (
+	"math/rand"
+	"time"
+)
+
+func MonteCarloPi(n int) float64 {
+	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
+	pass := 0
```

Comment:
On the other this is not the correct wording for this. I would call this variable something to signify that this is the inside of the unit circle.

Maybe it would be best to call this variable `circle` or `inside` not sure which one fits best. One thing I'm sure of is that `pass` is not a good name for this.

```suggestion
	circle := 0
```

Reply:
I chose the name inside